yarn.lock

```diff
@@ -7542,7 +7542,7 @@ map-visit@^1.0.0:
   dependencies:
     object-visit "^1.0.0"
 
-material-ui-search-bar@^1.0.0:
+material-ui-search-bar@1.0.0:
```

Comment:
We should __never__ edit auto generated files, like `yarn.lock`.

Reply:
Unless it is changed by running `yarn install` again, in which case I will resolve this comment :smile: